docs/Config_Reference.md

```diff
@@ -2458,6 +2458,10 @@ postfix for both sections.
 #   "calibration_extruder_temp" option is set.  Its recommended to heat
 #   the extruder some distance from the bed to minimize its impact on
 #   the probe coil temperature.  The default is 50.
+#max_valid_temp: 60.
+#   The maximum temperature used to validate the calibration.  It is
+#   recommended to set this to a value between 100 and 120 for enclosed
+#   printers.  The default is 60.
```

Comment:
This seems a bit confusing...

> This patch introduces the max_valid_temp config option, which replaces the hard coded maximum validation temperature of 120C.

Should the default not then be 120degC, such that existing installations are NOT impacted by this change?

And why have a recommendation of between 100 and 120deg, but then a default of 60deg?
shouldn't the default fall within the recommended range?  (I get that enclosed vs open will be different... but you haven't made this clarification in your description, even though you've obviously based the default 60deg value on it)

I think the name should also be changed to read 'max_validation_temp'.
'max_valid_temp' sounds like if the readback temperature exceeds this value then it will be considered as 'invalid'

Reply:
Existing calibrations won't be affected by this change, they have already been validated up to 120C.  The recommendation of 100-120C is for **enclosed** printers.    Defaulting to 60 makes it less likely users will fail validation after calibration.

I'm ok with changing the name for clarity.  In addition, I can tweak the post-calibration validation procedure a bit to use the maximum value between the `max_validation_temp` and the maximum measured temp.  This ensures that the entire range of measured temperatures is validated.